libgnucash/engine/qofinstance-p.h

```diff
@@ -166,13 +166,11 @@ void qof_instance_get_path_kvp (QofInstance *, GValue *, std::vector<std::string
 
 void qof_instance_set_path_kvp (QofInstance *, GValue const *, std::vector<std::string> const &);
 
-using StrVec = std::vector<std::string>;
-
 template <typename T> std::optional<T>
-qof_instance_get_path_kvp (QofInstance*, const StrVec&);
+qof_instance_get_path_kvp (QofInstance*, const std::vector<std::string>&);
```

Comment:
`qof_instance_get_path_kvp (QofInstance*, const Path&)`

`using Path = std::vector<std::string>>;` is in kvp-frame.hpp, which is included in qofinstance-p.h.